src/Badge.webmodeler.ts

```diff
@@ -30,11 +34,26 @@ export class preview extends Component<BadgeContainerProps, {}> {
             className: props.class,
             clickable: false,
             style: BadgeContainer.parseStyle(props.style),
-            value: valueAttribute ? "[" + valueAttribute + "]" : props.badgeValue
+            value: valueAttribute ? "[" + valueAttribute + "]" : props.badgeValue || "Badge"
```

Comment:
This is the incorrect place of doing a default value it should be in the widget.xml